types/unist/index.d.ts

```diff
@@ -6,13 +6,16 @@
 //                 Titus Wormer <https://github.com/wooorm>
 //                 Junyoung Choi <https://github.com/rokt33r>
 //                 Ben Moon <https://github.com/GuiltyDolphin>
+//                 JounQin <https://github.com/JounQin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
 /**
  * Syntactic units in unist syntax trees are called nodes.
+ *
+ * @typeParam TData Information from the ecosystem. Useful for more specific `Node#data`.
```

Comment:
I think this should be using `@link`.

```suggestion
 * @typeParam TData Information from the ecosystem. Useful for more specific {@link Node.data}
```

Reply:
Nice catch!